subsys/bluetooth/services/ots/otc.c

```diff
@@ -238,48 +271,48 @@ static void olcp_ind_handler(struct bt_conn *conn,
 
 	BT_DBG("OLCP indication");
 
-	if (op_code == BT_OTS_OLCP_PROC_RESP) {
-		enum bt_ots_olcp_proc_type req_opcode =
+	if (op_code == BT_GATT_OTS_OLCP_PROC_RESP) {
+		enum bt_gatt_ots_olcp_proc_type req_opcode =
 			net_buf_simple_pull_u8(&net_buf);
-		enum bt_ots_olcp_res_code result_code =
+		enum bt_gatt_ots_olcp_res_code result_code =
 			net_buf_simple_pull_u8(&net_buf);
 
 		print_olcp_response(req_opcode, result_code);
 
 		switch (req_opcode) {
-		case BT_OTS_OLCP_PROC_FIRST:
+		case BT_GATT_OTS_OLCP_PROC_FIRST:
```

Comment:
Outside the scope of this PR, but I don't think we really need the `GATT` infix for OTS defines. Could be renamed to `BT_OTS_`.

Reply:
It seems to be quite consequently used in OTS.  But I agree.
Would like to get some "blessing" from e.g. Johan in advance of changing all that.

Reply:
Can be postponed to another PR later. 

At least since OTS isn't in a `GATT` folder, the `BT_GATT` prefix doesn't make much sense to me at least :D